### DISCUSSION TITLE
How to extract streaming URLs?

### DISCUSSION THREAD

**Discussion by SwissTemp** [iptv-org/community#581] [Q&A] [answered]
Hi

I wanted to share what I found. 54 channels accessible from Switzerland with varying quality (432p to 720p). Maybe someone can extract the streaming URLs?

Just type embed-zattoo [dot] com / <channel_name>

sf-1
sf-2
3plus
4plus
ard
zdf
rtl
prosieben
sat1
rtl-2
vox
kabel-eins
orf-1
orf-2
sf-info
startv
s1
swiss_1
tv24
tv25
dmax
comedycentral
tlc
super-rtl
disney
nick
mtv
deluxe-music
eurosport
teleclub_zoom
tele1
tele-5
puls8
3sat
tf1
france-2
france-3
france-4
france-5
m6-suisse
tmc
gulli
nrj-12
tsr1
tsr2
rai-uno
rai-due
rai-tre
rete-4
canale-5
italia-1
la-7
rsi-la1
rsi-la2

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/discussions/new?category=channel-search
